minio/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for minio.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+secretKeyRefName: object-storage
+
+image:
+  repository: minio/minio
+  tag: latest
+  pullPolicy: IfNotPresent
+
+persistence:
+  storageClassName: gp2
+  accessMode: ReadWriteOnce
+  size: 30Gi
+
+ingress:
+  enabled: true
+  # Used to create an Ingress record.
+  hosts:
+    - minio.beta.k8s.jobteaser.net
```

Comment:
I am not ok with the default `host` in the ingress. This chart in not only for jobteaser.